Formula/unisonlang.rb

```diff
@@ -38,28 +39,56 @@ class Unisonlang < Formula
     version "M4e"
   end
 
+  resource "elm-git-install" do
+    url "https://registry.npmjs.org/elm-git-install/-/elm-git-install-0.1.4.tgz"
+    sha256 "f963ad9f8cc0d87c249cf8e63a62f648e3a3dd53ef658915c0062b2c27a11eeb"
+  end
+
+  resource "elm-json" do
+    url "https://github.com/zwilias/elm-json/archive/refs/tags/v0.2.13.tar.gz"
+    sha256 "8a6737e85e0a08ec3cc7de9efde3cf525fe0f0c3e7e5bb23796864edb09f31bc"
+  end
+
   def install
-    jobs = ENV.make_jobs
-    ENV.deparallelize
+    ENV["RUST_BACKTRACE"] = "1"
+    resource("elm-git-install").stage do
+      system "npm", "install", *Language::Node.std_npm_install_args(buildpath/"homebrew")
+    end
+    resource("elm-json").stage do
+      system "cargo", "install", *std_cargo_args(root: buildpath/"homebrew"), "--debug"
+    end
+    ENV.prepend_path "PATH", buildpath/"homebrew/bin"
 
     # Build and install the web interface
     resource("local-ui").stage do
       system "npm", "install", *Language::Node.local_npm_install_args
-      system "npm", "run", "ui-core:install"
-      system "npm", "run", "build"
 
+      # Manually run commands from `npm run ui-core:install`
+      Timeout.timeout(60) do
+        system "elm-git-install"
+      end
+      uicore = JSON.parse(File.read("elm-stuff/gitdeps/github.com/unisonweb/ui-core/elm.json"))
+      uicore["dependencies"].each do |name, version_range|
+        Timeout.timeout(60) do
+          system "elm-json", "-v", "-v", "-v", "install", "--yes", "#{name}@#{version_range.split(".")[0]}"
```

Comment:
Trying to add verbosity `-v -v -v`  output for debugging, but sadly making it harder to cause failure, which could be a side effect of slowing down output, serializing behavior on stdout, etc.

May need to try to reduce verbosity to see if I can repro.

For now,
- it seems to be an issue with `elm-json`
- happens on Big Sur or older (haven't yet reproduced on Monterey/Ventura). 
- fails with both pre-built binary and building it from source 
- WARN level logging (no `-v`) stalls without outputting anything
- TRACE level logging (`-v -v -v`) hasn't failed yet
- Haven't tried INFO (`-v`) or DEBUG (`-v -v`) yet.